### ISSUE TITLE
qt6-webengine

### ISSUE THREAD

**Issue by looooo** [conda-forge/qt-webengine-feedstock#42]
### Comment:

Are there builds for qt6-webengine available? Id not is it possible to use qt-webengine with qt6-main?

**Comment by ccordoba12** [comment]
> Are there builds for qt6-webengine available

No, there aren't. They are very difficult to compile so it's a hard job to do by volunteers.

> Id not is it possible to use qt-webengine with qt6-main?

No, it's not possible.

**Comment by h-vetinari** [comment]
There was an attempt in #46, now abandoned. This needs someone dedicated/motivated to debug...

**Comment by jschueller** [comment]
I'm retrying with #90 starting from the anaconda recipe, and it looks much more promessing than my previous attempt.

**Comment by h-vetinari** [comment]
This was closed by #90 🥳
